core/src/main/java/tachyon/master/MasterClient.java

```diff
@@ -843,8 +887,9 @@ public synchronized long worker_register(NetAddress workerNetAddress,
       connect();
 
       try {
-        long ret = mClient.worker_register(workerNetAddress, totalBytesOnTiers, usedBytesOnTiers,
-            currentBlockList);
+        long ret =
+            mClient.worker_register(workerNetAddress, totalBytesOnTiers, usedBytesOnTiers,
+                currentBlockList);
```

Comment:
Did the formatter change it to this format? I think the previous way was more condensed.